matrix/mat64/lq.go

```diff
@@ -58,10 +58,15 @@ func (lq *LQ) Factorize(a Matrix) {
 // TODO(btracey): Add in the "Reduced" forms for extracting the m×m orthogonal
 // and upper triangular matrices.
 
-// LFromLQ extracts the m×n lower trapezoidal matrix from a LQ decomposition.
-func (m *Dense) LFromLQ(lq *LQ) {
+// LTo extracts the m×n lower trapezoidal matrix from a LQ decomposition.
```

Comment:
Thanks for reminding me. Yes, we do - see comments https://github.com/gonum/matrix/pull/436#issuecomment-287593489.

I will go through the other instances that are like this and change them to match. For example https://github.com/gonum/matrix/pull/433#discussion_r107590374.